### PR TITLE
add NSIS mode

### PR DESCRIPTION
**Pull request by idleberg**

### CODE REVIEW

File: lib/ace/mode/nsis.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+var FoldMode = require("./folding/cstyle").FoldMode;
+
+var Mode = function() {
+    this.HighlightRules = NSISHighlightRules;
+    this.foldingRules = new FoldMode();
+};
+oop.inherits(Mode, TextMode);
+
+(function() {
+    // this.lineCommentStart = ""/\\*"";
```
Comment (nightwing) [inline lib/ace/mode/nsis.js]: this is needed for toggle comment command
looks like lineCommentStart should be `;` and blockComment = {start: "/*", end: "*/"};

**Comment by nightwing** [comment]
Looks good, could you please fill the  [The Individual CLA](https://docs.google.com/a/c9.io/forms/d/1MfmfrxqD_PNlNsuK0lC2KSelRLxGLGfh_wEcG0ijVvo/viewform) form? See https://github.com/ajaxorg/ace/blob/master/CONTRIBUTING.md for details

**Comment by idleberg** [comment]
Thank you, I signed and submitted the CLA.

By the way, do you also accept themes?

**Comment by nightwing** [comment]
Could you also fix lineCommentStart/blockComment? see https://github.com/ajaxorg/ace/pull/2802#discussion_r47138345

> By the way, do you also accept themes?

yes

**Comment by nightwing** [comment]
Thanks!

**Pull request merged**